libs/user_extensions/include/TTAlpsEvent.hpp

```diff
@@ -11,6 +11,11 @@ class TTAlpsEvent {
   auto Get(std::string branchName) { return event->Get(branchName); }
   float GetAsFloat(std::string branchName) { return event->GetAsFloat(branchName); }
   std::shared_ptr<PhysicsObjects> GetCollection(std::string name) const { return event->GetCollection(name); }
+  std::shared_ptr<PhysicsObjects> GetGenALPs();
+  std::shared_ptr<PhysicsObjects> GetGenMuonsFromALP();
+  std::shared_ptr<PhysicsObjects> GetMuonsFromALP(std::shared_ptr<PhysicsObjects> muonCollection, float maxDeltaR = 0.5);
```

Comment:
Not obvious what the difference is between `GetGenMuonsFromALP` and `GetMuonsFromALP` - normally muons for which we can check that they come from ALPs must be gen muons anyway. So I guess here among the muons provided as an argument the ones within maxDeltaR from the gen-muons coming from ALPs are returned. If that's the case, maybe a better name would be `GetMuonsMatchedToGenMuonsFromALP`. I know it's a bit long, but that would make it more clear.